etcdserver/api/v3rpc/lease.go

```diff
@@ -67,7 +72,8 @@ func (ls *LeaseServer) LeaseKeepAlive(stream pb.Lease_LeaseKeepAliveServer) erro
 			return err
 		}
 
-		resp := &pb.LeaseKeepAliveResponse{ID: req.ID, TTL: ttl}
+		resp := &pb.LeaseKeepAliveResponse{ID: req.ID, TTL: ttl, Header: &pb.ResponseHeader{}}
+		ls.hdr.fill(resp.Header)
```

Comment:
If the lease is revoked at `rev1`, does it matter that `resp.Header.Revision > rev1` is possible here?